src/pages/Landing/components/Description/index.jsx

```diff
@@ -0,0 +1,71 @@
+import styles from './styles.module.scss';
+import homeImage1 from '../../../../assets/images/landing/home-1.png';
+import homeImage2 from '../../../../assets/images/landing/home-2.png';
+import homeImage3 from '../../../../assets/images/landing/home-3.png';
```

Comment:
assets에서 가져오는 소스 파일들은 절대경로로 쓰는게 더 좋습니다~
```suggestion
import homeImage1 from '@/assets/images/landing/home-1.png';
import homeImage2 from '@/assets/images/landing/home-2.png';
import homeImage3 from '@/assets/images/landing/home-3.png';
```

Reply:
감사합니다~~~!